pkg/codegen/schema/schema.go

```diff
@@ -1997,6 +1997,8 @@ type BaseProviderSpec struct {
 	Name string `json:"name" yaml:"name"`
 	// The version of the base provider.
 	Version string `json:"version" yaml:"version"`
+	// The URL to use when downloading the provider plugin binary.
+	PluginDownloadURL string `json:"pluginDownloadURL,omitempty" yaml:"pluginDownloadURL,omitempty"`
```

Comment:
This shouldn't be here, it's already in the root of the schema. If the JSON says it should exist then remove it from the JSON schema.